src/CommonLib/Ntlm/HttpTransport.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SharpHoundCommonLib.Ntlm;
+
+public class HttpTransport : INtlmTransport
+{
+    private readonly ILogger _logger;
```

Comment:
`_logger` doesn't seem to be used, might not be needed, or potentially was forgot to be used